src/renderer/app-update-warning/app-update-warning.injectable.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import ipcRendererInjectable from "../app-paths/get-value-from-registered-channel/ipc-renderer/ipc-renderer.injectable";
+import sessionStorageInjectable from "../utils/session-storage.injectable";
+import { AppUpdateWarning } from "./app-update-warning";
+
+const appUpdateWarningInjectable = getInjectable({
+  id: "app-update-warning",
+
+  instantiate: (di) => {
+    AppUpdateWarning.resetInstance();
```

Comment:
Duuuude, `AppUpdateWarning` is a new global shared state, which we are trying to get rid of.